### PR TITLE
De-emphasize “GL” in the logo

### PR DESCRIPTION
**Pull request by AndrewKvalheim**
When tabling we’ve observed that “GL” tends to lead people to expect SeaGL to be a **g**raphics **l**ibrary. @altsalt had the idea of de-emphasizing it in the logo. I think it’s an improvement:

| Before | After |
|-|-|
| before | after |

Thoughts?

### CODE REVIEW

File: design/logo/seagl_avatar_profile_pic.png
Comment (AlfredoMS) [inline design/logo/seagl_avatar_profile_pic.png:1]: was the change in size from 600x600 to 512x512 intended?

Comment (AndrewKvalheim) [reply design/logo/seagl_avatar_profile_pic.png:1]: No, I matched the size of the assets that I’d created with the 2020 update and didn’t notice that this repository had diverged. I’ll update it to match this instead.

**Comment by altsalt** [comment]
This all looks good to me, no strong feelings on the 512x512 vs 600x600. Anything else to change before rolling this out? Let's make sure to update the remote ones too (if you haven't). Thanks!

**Comment by altsalt** [comment]
@AndrewKvalheim checking back in on this before merging. Again, I don't see any reason for the 600x600 vs 512x512, so go with your gut there. Could you also update the virtual versions of the logo?

**Pull request merged**